web/screenshot/app.po.ts

```diff
@@ -0,0 +1,8 @@
+import { browser, by, element } from 'protractor';
+
+export class AppPage {
+  navigateTo(): Promise<unknown> {
+    return browser.get(browser.baseUrl) as Promise<unknown>;
+  }
```

Comment:
Do we need this function?

Reply:
I just deleted the whole file and removed the import statement from `app.screenshot-spec.ts` and it works!